tests/__snapshots__/test_utils/test_bom_generation_altium_with_device_sheets.csv

```diff
@@ -21,13 +21,13 @@ GRM033R61A103KA01J,C1,b4654b650e69147ec39a6b967a45e02,1
 GRM155R61H104KE19D,C3,02cd4be3c70576c341d710602a673aa,1
 IHLP3232DZER100M01,L1,725321dda6339637d4d059423bc1c3c,1
 EEE-1HA470XP,C2,a00bbbee19879c290e62620ae7e47b1,1
-RC0805FR-071KL,R?,RES-2,1
-RC0402FR-077K5L,R?,RES,1
+RC0805FR-071KL,R2,RES-2,1
+RC0402FR-077K5L,R1,RES,1
 SBR1045D1-13,D1,3c54d775813915f30481cfffa5e9369,1
```

Comment:
This will be one of the cases I'll be looking at, since it triggered this initial investigation of the .annotations file: https://github.com/AllSpiceIO/py-allspice/pull/182/files#r1775928249

I expect this component will be assigned a new designator.

Reply:
Gotch, so it sounds like the test repo is "wrong" in the sense the board level annotation hasn't be generated correctly. @polypour can you regenerate the board level annotation for that design repo?

Reply:
I've updated the ref but there's another issue here: there's multiple components with the same UniqueID, and Altium seems to internally map them to other UniqueIDs, which means that the annotations don't apply for some of the components. I'm posting a detailed description to Notion because I can't see a path forward for that specific issue.

Reply:
https://www.notion.so/allspice/Altium-Read-Annotations-file-for-designators-in-BOM-11178a8a9a1980508f54d1029fbc099a?pvs=4#11178a8a9a198117a49cc65c764bbfd8

Reply:
This has been resolved in the latest commit, compare the test files with the Altium BOM samples in the repos.

Reply:
Nice, @polypour updated the test files with an Altium BOM export here: https://hub.allspice.io/AllSpiceTests/Altium-Device-Sheet-Usage-Demo/src/branch/main/DCDC%20Regulators%20Breakout/DCDC%20Regulators%20Breakout%20BOM%20Minimal.csv